lib/features/family/features/reflect/bloc/guess_secret_word_cubit.dart

```diff
@@ -52,9 +54,19 @@ class GuessSecretWordCubit
         0,
         1,
         2,
-        3
-      ]; // make sure people can't press wrong answers after pressing the correct answer
+        3,
+      ];
+
+      // make sure people can't press wrong answers after pressing the correct answer
```

Comment:
The comment placing is quite awkward now because the confetti is not the thing that fixes what the comment said.